pkg/api/postage.go

```diff
@@ -363,10 +365,18 @@ func (s *Service) reserveStateHandler(w http.ResponseWriter, _ *http.Request) {
 }
 
 // chainStateHandler returns the current chain state.
-func (s *Service) chainStateHandler(w http.ResponseWriter, _ *http.Request) {
+func (s *Service) chainStateHandler(w http.ResponseWriter, r *http.Request) {
+	logger := tracing.NewLoggerWithTraceID(r.Context(), s.logger)
 	state := s.batchStore.GetChainState()
-
+	chainTip, err := s.chainBackend.BlockNumber(r.Context())
+	if err != nil {
+		logger.Debugf("chainstate: block number: %v", err)
+		logger.Error("chainstate: block number unavailable")
+		jsonhttp.InternalServerError(w, nil)
+		return
+	}
 	jsonhttp.OK(w, chainStateResponse{
+		ChainTip:     chainTip,
```

Comment:
because this is an API change, the docs have to be updated, along with the openapi specs.

Reply:
@istae the property was in the docs but missing from the endpoint :shrug:

Reply:
https://docs.ethswarm.org/debug-api/#tag/Status/paths/~1chainstate/get